### PR TITLE
Wok 640 unittest fixes

### PR DESCRIPTION
**Pull request by VictorHarbo**
This PR adresses WOK-640 regarding fixing of old unittests and WOK-726 celanup of old code. I have done a combined cleanup on this branch

### CODE REVIEW

File: src/test/resources/xml/copyright_extraction/09222b40-dba1-11e5-9785-0016357f605f.xml [hunk 1]
```diff
@@ -45,17 +45,17 @@
                             <mods:languageTerm authority="rfc4646" type="code">da</mods:languageTerm>
                         </mods:languageOfCataloging>
                     </mods:recordInfo>
-                    <mods:relatedItem type="otherFormat">
-                        <mods:identifier displayLabel="image" type="uri">http://www.kb.dk/imageServicecumulus-core-01:/Depot/DAM/Samlingsbilleder/Samlingsbilleder/samlingsbilleder/KOB/Elfelt_4/Formidlingsfiler/Elf_113136.jpg</mods:identifier>
-                    </mods:relatedItem>
+                    <mods:titleInfo>
```
Comment (tokee) [inline src/test/resources/xml/copyright_extraction/09222b40-dba1-11e5-9785-0016357f605f.xml:48]: Shouldn't there be an image associated with this metadata record?

File: src/test/resources/xml/copyright_extraction/26d4dd60-6708-11e2-b40f-0016357f605f.xml [hunk 2]
```diff
@@ -57,14 +57,17 @@
                         <mods:topic lang="da">Undervisning</mods:topic>
                         <mods:topic lang="da">Kina</mods:topic>
                     </mods:subject>
+                    <mods:titleInfo lang="zh">
+                        <mods:title>?????????#Studér flittigt og gør fremskridt hver dag</mods:title>
```
Comment (tokee) [inline src/test/resources/xml/copyright_extraction/26d4dd60-6708-11e2-b40f-0016357f605f.xml:61]: What's up with the prefix `?????????#`?

**Review by tokee — CHANGES_REQUESTED** [review]
Minor suggestions and a few questions only.

File: src/test/java/dk/kb/present/transform/XSLTSolrTransformerTest.java [hunk 3]
```diff
@@ -63,25 +59,21 @@ void testSolrNew() throws Exception {
 	 */
 	@Test
 	void testXsltNewDpkItem() throws Exception {
-		String solrString = TestUtil.getTransformedWithAccessFieldsAdded(MODS2SOLR, RECORD_DPK);
+		String solrString = TestUtil.getTransformedWithAccessFieldsAdded(MODS2SOLR, RECORD_3956d820);
 		Gson gson = new GsonBuilder().setPrettyPrinting().create();
 		JsonElement je = JsonParser.parseString(solrString);
 		String prettyJsonString = gson.toJson(je);
```
[the remark this answers is not in the document]
Comment (VictorHarbo) [reply src/test/java/dk/kb/present/transform/XSLTSolrTransformerTest.java:65]: Helper method has been created. Some of the methods that doesnt test anything have been deleted as they were leftovers from earlier experimenting.

File: src/test/resources/xml/copyright_extraction/09222b40-dba1-11e5-9785-0016357f605f.xml (hunk 1 above)
Comment (VictorHarbo) [reply src/test/resources/xml/copyright_extraction/09222b40-dba1-11e5-9785-0016357f605f.xml:48]: Yes, ideally! However as I have just re-extracted the metadata record the field is still missing.

**Review by tokee — CHANGES_REQUESTED** [review]
Nice clean up. Only problem is that `XSLTSolrTransformerTest.testImageResource` does not pass.

**Review by tokee — APPROVED** [review]

**Pull request merged**